cg/test/random-test-db-environment.js

```diff
@@ -0,0 +1,37 @@
+const NodeEnvironment = require('jest-environment-node');
+
+function getRandomId() {
+  // Math.random should be unique because of its seeding algorithm.
+  // Convert it to base 36 (numbers + letters), and grab the first 9 characters
+  // after the decimal.
+  return Math.random()
+    .toString(36)
+    .substr(2, 9);
+}
+
+module.exports = class RandomDbEnvironment extends NodeEnvironment {
+  constructor(config) {
+    super(config);
+  }
+
+  async setup() {
+    await super.setup();
+
+    this.global.dbName = `${process.env.DB_DATABASE}_${getRandomId()}`;
```

Comment:
This global is only set for tests, otherwise it will be read from ENV vars (hack continued)